HidInjector/app/HidInjectorTest.c

```diff
@@ -222,11 +207,11 @@ SendTestInput(
 		InjectKeyDown(VK_RETURN);
 		InjectKeyUp(VK_RETURN);
 	}
+    */
```

Comment:
I'll add a note. These are how you'd send these messages, but doing them all together they really make no sense.